### PR TITLE
Remove expensive call to scan

### PR DESCRIPTION
**Pull request by arturictus**
In issue #69 @Fire-Dragon-DoL pointed out that we were doing an expensive call to redis with the only purpose to print information to the stdout.

Reviewing the code I realised that we had functionality from the previous implementation that was not used anymore I removed that code

### CODE REVIEW

**Comment by Fire-Dragon-DoL** [comment]
Looks good! :+1:

**Review by Fire-Dragon-DoL — APPROVED** [review]

**Review by andrcuns — APPROVED** [review]
Looks good to me 👍 

There is a rubocop failure, but I personally don't really agree with that rule. So maybe we need to update the .rubocop.yml. But You can also fix the violation if You like.

File: lib/sidekiq_alive/worker.rb [hunk 1]
```diff
@@ -5,20 +5,15 @@ class Worker
...
       write_living_probe
       # schedules next living probe
-      self.class.perform_in(config.time_to_live / 2, current_hostname)
-    end
-
-    def hostname_registered?(hostname)
-      SidekiqAlive.registered_instances.any? do |ri|
-        /#{hostname}/ =~ ri
-      end
+      self.class.perform_in(config.time_to_live / 2)
```
Comment (arturictus) [inline lib/sidekiq_alive/worker.rb:16]:
```suggestion
      self.class.perform_in(config.time_to_live / 2, current_hostname)
```

File: lib/sidekiq_alive/worker.rb [hunk 2]
```diff
@@ -5,20 +5,15 @@ class Worker
     include Sidekiq::Worker
     sidekiq_options retry: false
 
-    def perform(_hostname = SidekiqAlive.hostname)
+    # Passing the hostname argument it's only for debugging enqueued jobs
+    def perform(_hostname = nil)
```
Comment (arturictus) [inline lib/sidekiq_alive/worker.rb:9]:
```suggestion
    def perform(_hostname)
```

File: lib/sidekiq_alive.rb [hunk 3]
```diff
@@ -135,7 +139,7 @@ def startup_info
     end
 
     def successful_startup_text
-      "Successfully started sidekiq-alive, registered instances: #{registered_instances.join("\n\s\s- ")}"
+      "Successfully started sidekiq-alive, register with key: #{current_instance_register_key}"
```
Comment (arturictus) [inline lib/sidekiq_alive.rb:142]:
```suggestion
      "Successfully started sidekiq-alive, registered with key: #{current_instance_register_key}"
```

**Pull request merged**